recipe/meta.yaml

```diff
@@ -40,7 +40,10 @@ requirements:
     - packaging
   run_constrained:
     - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx]
-    - pytorch >=1.4,<1.7
+    - pytorch >=1.4,<1.7  # inferred by testing
+    - scikit-learn >=0.17,<=0.19.2
+    - tensorflow >=1.12.0,<=1.15.0|>=2.1.0,<=2.3.1
```

Comment:
TF 1.11.0 should also work I believe.

Reply:
I think it was bumped up to 1.12 with the coremltools 4.0 release, assuming this file does what I think it does:

https://github.com/apple/coremltools/blob/master/coremltools/_deps/__init__.py#L89